Cargo.toml

```diff
@@ -88,6 +88,7 @@ constant_time_eq = "0.2.4"
 rayon = { version = "1.2.1", optional = true }
 cfg-if = "1.0.0"
 digest = { version = "0.10.1", features = [ "mac" ], optional = true }
+zeroize = { version = "1", features = ["zeroize_derive"], optional = true }
```

Comment:
I think we need `default-features = false` here, so that this doesn't pull in `alloc`? And then we should also make our own `std` feature above pull in `zeroize/std` (just like it's currently doing with `digest/std`). Does that sound right?

Reply:
That will enable this also when `zeroize` is not enabled (that's actually what happens now with `digest`)
if we want to do this (which I'm not sure we want, why should you enable `std` on a crate if you don't use anything it adds?) we need the weak dependencies feature which was stabilized in 1.60 (https://github.com/rust-lang/cargo/issues/10269) this will allow us to do `std = [zeroize?/std]` which will only enable the feature of the dep if the dep is actually enabled

Reply:
Ah that's a good point. The `std` features of those crates are independent of what we're deriving, and the caller can depend on that feature themselves if they need it. Do you think `default-features = false` still makes sense?

Reply:
Yes, I'll add that